.travis.yml

```diff
@@ -39,7 +39,7 @@ before_script:
 #  - chmod +x travis-acceptance-tests.sh
```

Comment:
I think we can remove the `before_script` section, since we aren't running Espresso tests on the emulator anymore (and the relevant line is commented out anyway).